scripts/monit_start_service.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+#
+import logging
+import os
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../AppDB'))
+from cassandra_env import cassandra_interface
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../lib'))
+from constants import APPSCALE_HOME
+from zkappscale import zktransaction as zk
+import monit_app_configuration
+import monit_interface
+
+# The location of the Cassandra binary on the local filesystem.
+CASSANDRA_EXECUTABLE = cassandra_interface.CASSANDRA_INSTALL_DIR \
+  + "/cassandra/bin/cassandra"
+
+# The location on the local file system where we write the process ID
+# that Cassandra runs on.
+PID_FILE = "/var/appscale/appscale-cassandra.pid"
+
+# The default port to connect to Cassandra.
+CASSANDRA_PORT = 9999
+
+# Monit watch name for Cassandra.
+CASSANDRA_WATCH_NAME = "cassandra"
```

Comment:
Please reuse the watch name constants defined in datastore_upgrade.py if possible.